wagtail/test/testapp/wagtail_hooks.py

```diff
@@ -390,3 +391,9 @@ class DisableBulkAction(SnippetBulkAction):
 @hooks.register("register_admin_viewset")
 def register_animated_advert_chooser_viewset():
     return animated_advert_chooser_viewset
+
+
+@hooks.register("register_admin_search_test")
```

Comment:
Not sure where you got this but this isn't a valid Wagtail hook. To register a `SnippetViewSet`, you need to use `register_snippet`, e.g.


```py
register_snippet(SnippetSearchTestModelViewSet)
```

Reply:
Sorry for this mistake @laymonage, i mixed up the something like 
```
@hooks.register("register_admin_viewset")
def register_viewset():
    return YourViewSet()
```
Nonetheless, i fixed it and testcases are passing. Thank you for the help